### PR TITLE
[Notifier] [RocketChat] Add tests and pull up scheme check

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 5.1
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | ---
| License       | MIT
| Doc PR        | ---

Replaces #39445 (because it targetted the wrong branch)

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/RocketChat/phpunit.xml.dist [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <testsuite name="Symfony RocketChat Notifier Bridge Test Suite">
+            <directory>./Tests/</directory>
+        </testsuite>
+    </testsuites>
+
+    <filter>
+        <whitelist>
+            <directory>./</directory>
+            <exclude>
+                <directory>./Resources</directory>
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/RocketChat/phpunit.xml.dist:25]: @nicolas-grekas none of the bridges have a Resources folder, shall we remove it from all phpunit files?

Comment (derrabus) [reply src/Symfony/Component/Notifier/Bridge/RocketChat/phpunit.xml.dist:25]: As long as PHPUnit does not complain, I'd leave the entries in here. We might add such a folder in the future and re-adding the line then is something that might get overlooked easily then.

Comment (OskarStark) [reply src/Symfony/Component/Notifier/Bridge/RocketChat/phpunit.xml.dist:25]: OK, thanks for the feedback!

**Comment by OskarStark** [comment]
I cherry picked the code of this PR to #39455

cc @jderusse

**Pull request closed without merging**